main.py

```
@@ -8139,20 +8139,34 @@ async def roll(message: discord.Interaction, sides: Optional[int]):
-            side = "heads"
-        await message.response.send_message(f"🪙 your coin lands on **{side}** ({coinflipresult})")
-    else:
+    view = View(timeout=VIEW_TIMEOUT)
+    button = Button(label="Reroll", emoji="🎲", style=ButtonStyle.blurple)
+
+    async def roll_and_respond(interaction: discord.Interaction, is_first=False):
+        if interaction.user != message.user:
+            await do_funny(interaction)
+            return
+
         roll = random.randint(1, sides)
```

Ruff (0.15.20)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)